checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleProfileImporter.java

```diff
@@ -43,6 +45,9 @@
 
 import com.google.common.annotations.VisibleForTesting;
 
+@ExtensionPoint
+@ScannerSide
+// Note: RuleFinder is deprecated and should be replaced.
```

Comment:
please remove this note and create Issue on this.
We do not need such comments, that are useless, as they does not trigger engineers to make an action.

Reply:
moved to https://github.com/checkstyle/sonar-checkstyle/issues/185